app/(app)/settings/index.tsx

```diff
@@ -32,6 +35,16 @@ const SettingsPage = () => {
     [],
   )
 
+  const { mutate, isPending } = useMutation({
```

Comment:
I would follow the style as on other mutations, to use whole mutation object instead of destructuring. What do you think?